### ISSUE TITLE
admin: ServiceStatus should return back server uptime 

### ISSUE THREAD

**Issue by harshavardhana** [minio/minio#3678]
```
$ mc admin service status play
Total: 630GiB, Free: 561GiB. Online Disks: 8, Offline Disks: 0.

$ mc admin service status play --json
{"status":"success","service":true,"storageInfo":{"total":675875831808,"free":602565951488,"backend":{"backendType":"XL","onlineDisks":8,"offlineDisks":0}},"server":{"version":"2017-02-02T20:07:53Z","commitID":"aacfb8654ee399922ca70a6f2d86e23e84ce39e5"}}

``` 
## Expected Behavior
Server should report back the server uptime i.e  duration since when the server is online.

## Current Behavior
Do not know since when the server is up.

## Possible Solution
Remember the time when the server has been started after object layer initialization.

## Context
Usability improvements.

## Your Environment
Current `mc master` and `minio master`

**Comment by harshavardhana** [comment]
Change output structure of `service status`

```
$ mc admin service status play/ --json | jq .
{
  "status": "success",
  "service": true,
  "storageInfo": {
    "total": 675875831808,
    "free": 602565951488,
    "backend": {
      "backendType": "XL",
      "onlineDisks": 8,
      "offlineDisks": 0
    }
  },
  "server": {
    "version": "2017-02-02T20:07:53Z",
    "commitID": "aacfb8654ee399922ca70a6f2d86e23e84ce39e5"
  }
}
```

To

```
$ mc admin service status play/ --json | jq .
{
  "status": "success",
  "service": "online|offline",
  "uptime": 0000,
}
```

**Comment by GarimaKapoor** [comment]
@krisis please update the timeline for the fix.

**Comment by krisis** [comment]
@harshavardhana what is the uptime we should report in the case of a distributed setup?

**Comment by harshavardhana** [comment]
> @harshavardhana what is the uptime we should report in the case of a distributed setup?

Quorum based commonality as discussed.

**Comment by vadmeste** [comment]
@krisis, uptime can be wrongly calculated in case of having system's time changed by the user or NTP, it seems to me that currently, there is no way to properly address this but it will be *transparently* fixed in go 1.9 (see https://github.com/golang/go/issues/12914 and https://github.com/golang/proposal/blob/master/design/12914-monotonic.md)

**Comment by krisis** [comment]
@GarimaKapoor the fix for this issue is available for review here - #3690
